juniper_official/Node-Exporter/check-cpu-usage.rule

```diff
@@ -0,0 +1,164 @@
+/*
+ * Checks if cpu usage is exceeding threshold value and raises alarm.
+ * 
+ * Two input control detection.
+ * 
+ *   1) "threshold" is used to check if the used percentage is
+ *      increasing by more than the threshold value.
+ *   2) "freq" is the interval between data points.
+ *  
+ */
+healthbot {
+    topic node.exporter {
```

Comment:
Can we come up with a more intuitive topic name? For instance the original ingest name which was node-exporter was changed to server-monitoring. 
Maybe we go with "server monitoring" "server stats" or "host server" or some such. @gvarao - thoughts?

Reply:
Changed to Server Monitoring.